docs/releasing.md

```diff
@@ -96,6 +96,12 @@ machine and update the release branch:
 git fetch && git reset --hard $LK_REMOTE/rel_$LK_RELEASE
 ```
 
+Finally, with all of the arch-specific images out, push out the multi-arch index:
+
+```sh
+make push-manifest
+```
+
 Stash the tag of the alpine base image in an environment variable:
 
 ```sh
```

Comment:
This also needs changing. Best to replace with `make show-tag`

Reply:
Good catch. Updated.